### ISSUE TITLE
Update PyPI package

### ISSUE THREAD

**Issue by pierresegonne** [Breakend/experiment-impact-tracker#62]
Hey,

It seems like the PyPI package is not up to date at all anymore with the code and the documentation. 

Notably, the `ExperimentTracker` class has no `__enter__` method, resulting in a crash when trying to use the implementation,

```python
with ExperimentTracker(..):
```

**Comment by Omerkvc** [comment]
> Hey,
> 
> Görünüşe göre PyPI paketi, kod ve belgelerle artık hiç güncel değil.
> 
> Özellikle, `ExperimentTracker`sınıfın hiçbir `__enter__`yöntemi yoktur, bu da uygulamayı kullanmaya çalışırken bir çökmeye neden olur.
> 
> ```python
> with ExperimentTracker(..):
> ```

hello, now i'm trying to execute experiment impact tracker but I take errors, it can be reletad python's version ? if you can use experiment tracker without errors, which python version are you using ? thanks

**Comment by luciabouzaheguerte** [comment]
Hi. when using pip install, version installed is 0.1.8 (witch has not DataInterface, for example).
Version in master is 0.1.9

**Comment by JingdiZhou** [comment]
> 

Yes, the version 0.1.8 has no DataInterface ,which confused me for a long time in my code until I discovered this problem, but later I added the file data_interface.py to the conda path of experiment-impact-tracker and finally it worked
